locator/locator.go

```diff
@@ -102,20 +103,24 @@ func getInterfaceFromImportPath(interfaceName, importPath string) (*model.Interf
 			return nil, err
 		}
 
-		typeNames := getTypeNames(pkg)
-
 		if iface != nil {
-			var methods []*ast.Field
-			var err error
-			var imports []*ast.ImportSpec = getImports(file)
+			typesFound := getTypeNames(pkg)
+			importSpecs := getImports(file)
 
+			pkgImport := pkg.Name
+			if strings.HasSuffix(importPath, pkg.Name) {
+				pkgImport = "xyz123"
```

Comment:
Seems like the intent here is to de-dupe the name, or reduce collisions?

This should really be tracked so we can make the import names be meaningful (e.g.: `pkg1`, then `pkg2` and so on...), right?

Would love to address this one tiny bit before merging.